view/frontend/web/js/view/payment/method-renderer/omise-offline-tesco-method.js

```diff
@@ -77,23 +35,17 @@ define(
              */
             placeOrder: function(data, event) {
                 var self = this;
+                var failHandler = this.buildFailHandler(self);
```

Comment:
In a case where we encapsulate everything inside `this.buildFailHandler` now. 
Do we still need `var self = this` and all the `self` variables here?

Reply:
👍